routes/api_v1_0.py

```diff
@@ -210,35 +274,124 @@ def scale_v1_0(dataset: str, variable: str, scale: str):
     Returns a scale bar
     """
 
-    return routes.routes_impl.scale_impl(dataset, variable, scale)
+
+    bytesIOBuff = plotting.tile.scale({
+        'dataset': dataset,
+        'variable': variable,
+        'scale': scale,
+    })
+
+    return send_file(bytesIOBuff, mimetype="image/png", cache_timeout=MAX_CACHE)
 
 
 @bp_v1_0.route('/api/v1.0/range/<string:dataset>/<string:variable>/<string:interp>/<int:radius>/<int:neighbours>/<string:projection>/<string:extent>/<string:depth>/<int:time>.json')
 def range_query_v1_0(dataset: str, variable: str, interp: str, radius: int, neighbours: int, projection: str, extent: str, depth: str, time: int):
-    return routes.routes_impl.range_query_impl(interp, radius, neighbours, dataset, projection, extent, variable, depth, time)
+    extent = list(map(float, extent.split(",")))
+
+    min, max = plotting.scale.get_scale(
```

Comment:
CodeFactor has a good point here...could we rename these variables to `min_data_val` and `max_data_val` or something similar? I really don't like interfering with built-in `min` and `max` functions.